docs/source/vagrant_variables.yml.example

```diff
@@ -0,0 +1 @@
+../../vagrant_variables.yml.example
```

Comment:
@Zeitsperre What was this symlink for again?

Reply:
Ah, it was to address the file access problem. I was exploring symlinking into the docs folder all the files I needed, but looks like you took a different approach. Looking at it all now.

Reply:
My other "approach" only works for files in `birdhouse/` folder only, all other files still need the "download links" trick.  So no nothing new, no change of direction.

Reply:
It would be interesting if we could use hard links both in  `/birdhouse`  and under `/docs/source/`, but that might cause issues in other ways.

Reply:
> It would be interesting if we could use hard links both in `/birdhouse` and under `/docs/source/`, but that might cause issues in other ways.

Not sure how this is going to solve our relative links on Github not working on RtD problem.   What we really need is either

* a way to copy files into the html output folder at any location we want (the option I found in db4d5aab1bf24a8d2cd9972b2e0a638d35533e5f only copy to the root of the html output folder)

* a way to instruct RtD/Sphinx to rewrite all relative links to absolute link to Github at the same changeset of the build on RtD (ex: a relative link like `../env.local.example` should be replace with `https://github.com/bird-house/birdhouse-deploy/blob/COMMIT_HASH_OF_RTD_BUILD/birdhouse/env.local.example`